doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+**Pros**
+- Easy to explain as rules are simple to interpret
+- Is a black-box method as we just need to predict the value of an instance and don't need to access model internals
+- Can be parallelized and made to be much more efficient as a result
+- Although they apply to a local instance, the notion of coverage also gives a level of global insight
+
+**Cons**
+- This algorithm is much slower for high dimensional feature spaces
+- If choosing an anchor close to a decision boundary, the method may require a considerable number of samples from $\mathcal{D}(z|A)$ and $\mathcal{D}(z|A')$ to distinguish two anchors $A$ and $A'$.
+- High dimensional feature spaces such as images need to be reduced. Typically, this is done by segmenting the image into superpixels. The choice of the algorithm used to obtain the superpixels has an effect on the anchor obtained.
+- Practitioners need to make several choices concerning parameters and domain-specific setup. For instance, the precision threshold or the method by which we sample $\mathcal{D}(z|A)$. Fortunately, alibi provides default settings for a lot of specific data types.
+
+
+#### Pertinent Positives
+
+| Model-types | Task-types     | Data-types  |
+| ----------- | -------------- | ----------- |
+| Black-box   | Classification | Tabular     |
+|             |                | Image       |
+
+
+Informally a Pertinent Positive is the subset of an instance that still obtains the same classification. These differ from anchors primarily in the fact that they aren't constructed to maximize coverage. The method to create them is also substantially different. The rough idea is to define an absence of a feature and then perturb the instance to take away as much information as possible while still retaining the original classification.
+
+Given an instance $x_0$ we set out to find a $\delta$ that minimizes the following loss:
+
+$$
+L = c\cdot L_{pred}(\delta) + \beta L_{1}(\delta) + L_{2}^{2}(\delta) + \gamma \|\delta - AE(\delta)\|^{2}_{2}
+$$
+
+where
+
+$$
+L_{pred}(\delta) = max\left\{ max_{i\neq t_{0}}[Pred(\delta)]_{i} - [Pred(\delta)]_{t_0}, \kappa \right\}
+$$
+
+$AE$ is an optional autoencoder generated from the training data. If delta strays from the original data distribution, the autoencoder loss will increase as it will no longer reconstruct $\delta$ well. Thus, we ensure that $\delta$ remains close to the original dataset distribution.
```

Comment:
- I don't think AE is optional in the CEM method, it really is the key ingredient.
 - delta --> \delta

Reply:
Ah, both the paper and the [docs](https://docs.seldon.io/projects/alibi/en/latest/api/alibi.explainers.cem.html#module-alibi.explainers.cem) list the AE as optional?
![CEM-method](https://user-images.githubusercontent.com/29690500/142462060-fce324c3-ad41-435c-b959-453eb12d6b22.png)
I confess this surprised me as without the autoencoder it doesn't seem very different to the counterfactual instances method except that the loss is slightly different and they see this idea of absence or presence of features.

Reply:
Yea for CEM imo it shouldn't be really optional, but I can see how you might make a case. I don't think it's too big of a point to spend time on, but I would personally remove the "optional" because it might confuse things more than clarify.